src/components/Switch/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import './style.css';
+
+class Switch extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            chosenParamIndex: 0,
+        };
+    }
+    handleClick(which) {
+        this.setState({chosenParamIndex: which })
```

Comment:
try to better format your code... at least add space after opening curved bracket and before closing curved bracket

Reply:
fixed